srtcore/queue.h

```diff
@@ -429,6 +429,10 @@ friend class CUDTUnited;
    } m_WorkerStats;
 #endif /* SRT_DEBUG_SNDQ_HIGHRATE */
 
+#if ENABLE_LOGGING
+   static int m_counter;
+#endif
```

Comment:
It may be better to move this under heavy logging.

Reply:
Again, I don't have `IF_LOGGING`, only `IF_HEAVY_LOGGING`.

Reply:
I mean the whole counter thing

Reply:
Well, this is used exclusively with spelling up names for the threads. It's not counting any execution or something of the kind. This is so that the thread names are set as `SRT:SndQ:w5` or alike.